src/foremast/utils/templates.py

```diff
@@ -47,6 +48,7 @@ def get_template_object(template_file=''):
 
     if TEMPLATES_PATH:
         external_templates = os.path.expanduser(TEMPLATES_PATH)
+        external_templates = str(pathlib.Path(TEMPLATES_PATH))
```

Comment:
This is missing the previous `.expanduser()` behaviour. It also sounds like you want to start using `.resolve()`.

Reply:
Good point. fixed that.